haystack/components/converters/azure.py

```diff
@@ -56,8 +56,8 @@ def __init__(self, endpoint: str, api_key: Optional[str] = None, model_id: str =
         self.endpoint = endpoint
         self.model_id = model_id
 
-    @component.output_types(documents=List[Document], azure=List[Dict])
-    def run(self, sources: List[Union[str, Path, ByteStream]]):
+    @component.output_types(documents=List[Document], raw_azure_response=List[Dict])
```

Comment:
this is not related to this PR but the declaration did not match with the return value of the `run` method, so I fixed it.